src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskProviderTest.java

```diff
@@ -160,7 +161,7 @@ private Task createTask() {
                         PermissionTypes.MANAGE,
                         PermissionTypes.REFER
                     ))),
-            "LEGAL_OPERATIONS",
+            RoleCategory.LEGAL_OPERATIONS.name(),
```

Comment:
can we also update the local consumer to include role_category?

TaskManagerGetTaskConsumerTest